.devops/kubernetes/helmCharts/templates/deployment.yml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hub-spid-login-ms
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: hub-spid-login-ms
+  template:
+    metadata:
+      labels:
+        app: hub-spid-login-ms
+    spec:
+      containers:
+        - name: hub-spid-login-ms
+          image: {{ .Values.containerRegistry }}/{{ .Values.imageRepository }}
+          imagePullPolicy: Always
+          ports:
+            - containerPort: {{ .Values.port }}
```

Comment:
I don't see the Values file for the helm chart, is correct?

Reply:
values are injected in the command line: this is configured inside the azure pipeline

Reply:
https://github.com/pagopa/hub-spid-login-ms/blob/SELC-10_k8s-deploy/.devops/kubernetes/deploy-pipelines.yml#L118

Reply:
Please create a `values.yaml` with the default values to understand better how to manage the variables